include/arch/arm/arch/machine/gic_v2.h

```diff
@@ -124,13 +124,14 @@ struct gic_cpu_iface_map {
 
 extern volatile struct gic_dist_map *const gic_dist;
 extern volatile struct gic_cpu_iface_map *const gic_cpuiface;
+volatile uint32_t gic_dist_config_cache[32];
```

Comment:
nit: Maybe #define the 32 as something. It isn't #defined in the original gic_dist, but when its not in the struct, it feels like it shouldn't be a raw number, idk.

Reply:
Also I don't think it needs to be volatile, cos its not tied to hardware anymore so it wont randomly change.

Reply:
Also, I think its meant to be 64